.github/workflows/release.yml

```diff
@@ -30,4 +30,4 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GH_RELEASE_BUILDUSERSICKAG }}
           NPM_TOKEN: ${{ secrets.NPM_RELEASE_TOKEN_CS }}
-        run: pnpm build.release && pnpm publish --tag main
+        run: pnpm release && pnpm publish --recursive --tag main
```

Comment:
I think we will still have to do a `build.release` as we deploy the `dist` files